### DISCUSSION TITLE
r-peaks detection don't work

### DISCUSSION THREAD

**Discussion by manef1994** [neuropsychology/NeuroKit#649] [Q&A]
I'm using the neurokit2 library to detect R peaks. For some acquisitions, after applying " rodrigues2021 " algorithm I got this error message which couldn't fix it:

if processed_ecg[i] > Rpeakamp:
IndexError: index 2053620 is out of bounds for axis 0 with size 2053620

is there any solution for this problem ?

**Comment by DominiqueMakowski** [comment]
duplicate of #555
